### PR TITLE
Drop Google Play Music-related packages

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
The service was discontinued.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dotlambda** [comment]
For some reason, this breaks `beetsExternalPlugins.check`.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 115239` at 5ae8cdb4 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/df971f125292a6cb012670b8a2a3fbfe">beetsExternalPlugins.alternatives</a></li></li>
    <li><a href="https://gist.github.com/2e745257a34e1c14f4c52722db877024">beetsExternalPlugins.check</a></li></li>
  </ul>
</details>
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/251cbb3b888503f8bb0e8d49ba6533d2">beets</a></li></li>
    <li><a href="https://gist.github.com/0580d104c7b54cd6b0b3048eeb0b455b">beetsExternalPlugins.copyartifacts</a></li></li>
    <li><a href="https://gist.github.com/1f4658a3bcbc99f6cf73d0913c4455b3">beetsExternalPlugins.extrafiles</a></li></li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
> For some reason, this breaks `beetsExternalPlugins.check`.

It is already failing on master.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
we need aliases for all programs removed.

**Comment by dotlambda** [comment]
> It is already failing on master.

Oops, I meant `beetsExternalPlugins.alternatives`.

**Comment by dotlambda** [comment]
> we need aliases for all programs removed.

Are you sure? I think anyone using them will know by now that the service was discontinued.

**Comment by doronbehar** [comment]
> > we need aliases for all programs removed.
> 
> Are you sure? I think anyone using them will know by now that the service was discontinued.

Still, these packages might get evaluated somewhere but not used.

> > > For some reason, this breaks `beetsExternalPlugins.check`.
> >
> > It is already failing on master.
> 
> Oops, I meant `beetsExternalPlugins.alternatives`.

Just checked, and `alternatives` doesn't fail on master but `check` does. An update to alternatives might help:

https://github.com/geigerzaehler/beets-alternatives/releases

And although it's not directly related to this PR, marking `beetsExternalPlugins.check` as broken would be nice.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -2766,12 +2764,6 @@ in {
 
   googlemaps = callPackage ../development/python-modules/googlemaps { };
 
-  google-music = callPackage ../development/python-modules/google-music { };
-
-  google-music-proto = callPackage ../development/python-modules/google-music-proto { };
-
-  google-music-utils = callPackage ../development/python-modules/google-music-utils { };
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix:2773]: some entries in aliases.nix would be nice to have.

**Comment by dotlambda** [comment]
Now `beetsExternalPlugins.alternatives` works again, i.e. it builds and no tests fail. I didn't test functionality though.

**Review by lovesegfault — APPROVED** [review]
Beets changes LGTM

**Review by aszlig — APPROVED** [review]

**Review by doronbehar — APPROVED** [review]

**Comment by dotlambda** [comment]
> I used Google Play Music Desktop Player to access YouTube Music (as it has support for the service). Can this package be reinstated?

https://github.com/NixOS/nixpkgs/pull/115907

**Pull request merged**